selfie.dev/src/pages/py/get-started.mdx

```diff
@@ -1,47 +1,31 @@
 import { DocsImage } from "@/components/DocsImage";
 
 export const title = "Selfie Python Snapshot Testing | Get started";
-export const description = "Zero-config inline and disk snapshots for Python. Add a single dependency and you're all set.";
+export const description =
+  "Zero-config inline and disk snapshots for Python. Add a single dependency and you're all set.";
 export const imageUrl = "https://selfie.dev/get-started.webp";
 
 <DocsImage imgAbsoluteUrl={imageUrl} />
 
 TODO: adapt this to Python
 
-To start snapshot testing in Java or any JVM language, all you need is to add a [single dependency](https://central.sonatype.com/artifact/com.diffplug.selfie/selfie-runner-junit5).
+To start snapshot testing in Python, all you need is to add a [single dependency](https://central.sonatype.com/artifact/com.diffplug.selfie/selfie-runner-junit5).
 
 ## Installation
 
 ### Requirements
 
-Selfie snapshot testing works with the following JVM test runners:
+Selfie snapshot testing works with the following Python test runners:
 
-- JUnit5
-- JUnit4 through junit-vintage
-- [Kotest (with JUnit5 or Kotlin Multiplatform)](./kotest)
+- Pytest
 - PRs welcome for other test runners
 
-Disk snapshots can be used with any JVM language, but inline literal snapshots only work if the test code is written in:
+Both disk and inline snapshots can be used with Python test code.
 
-- java (8+ is okay, but 15+ is recommended)
-- kotlin
-- groovy
-- scala
+### Poetry
 
-Adding support for other languages is [straightforward](https://github.com/diffplug/selfie/blob/main/jvm/selfie-lib/src/commonMain/kotlin/com/diffplug/selfie/guts/Literals.kt), PRs are welcome!
-
-### Maven
-
-If you're using Maven, add the following dependency to your `pom.xml` file:
-
-```xml
-<dependency>
-    <groupId>com.diffplug.selfie</groupId>
-    <artifactId>selfie-runner-junit5</artifactId>
-    <version>${ver_SELFIE}</version>
-    <scope>test</scope>
-</dependency>
-```
+Dependencies are managed using [poetry](https://python-poetry.org/docs/#installing-with-the-official-installer),
+then just cd into `selfie-lib` and `run poetry install`.
 
 Replace `ver_SELFIE` with the [latest available version of selfie](https://github.com/diffplug/selfie/blob/main/jvm/CHANGELOG.md).
```

Comment:
Are we going to have an update log to link as well? should I unlink for now?

Reply:
Make a changelog and make a link, copy the format of the jvm one.